mifosng-provider/src/main/java/org/mifosplatform/portfolio/calendar/service/CalendarUtils.java

```diff
@@ -111,13 +111,11 @@ private static Date convertToiCal4JCompatibleDate(final LocalDate inputDate) {
     private static Collection<LocalDate> convertToLocalDateList(final DateList dates) {
 
         final Collection<LocalDate> recurringDates = new ArrayList<>();
```

Comment:
please make the calendar changes a seperate pull request. Best to have one pr correspond to one issue on JIra